neptune/new/internal/init/model.py

```diff
@@ -59,16 +56,6 @@ def init_model(
     flush_period: float = DEFAULT_FLUSH_PERIOD,
     proxies: Optional[dict] = None,
 ) -> Model:
-    if model:
-        if with_id:
-            raise NeptuneInitParametersCollision("with_id", "model", method_name="init_model")
-
-        _logger.warning(
-            "parameter `model` is deprecated, use `with_id` instead."
-            " We'll end support of it in `neptune-client==1.0.0`."
-        )
-        with_id = model
```

Comment:
I think something like `with_id = with_id or model` is still necessary.

Reply:
Decorator is responsible for passing deprecated kwarg as `with_id` to function.